Source/Cmlx/mlx-generated/arange.cpp

```diff
@@ -0,0 +1,17 @@
+namespace mlx::core::metal {
```

Comment:
These are the kernels that we will jit -- see `tools/update-mlx.sh` which uses CMake to build these